docker/tests/Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN chmod +x /start.sh
 COPY ./docker/tests/check-code.sh /check-code.sh
 RUN chmod +x /check-code.sh
 
+RUN python3 -m pip install -U pip pytest

```

Comment:
Following the pattern from `nose`, I think it fits better with existing infra to have this listed in the dev requirements, and installed from there rather than a separate command in the dockerfile.